riak_repl/CHANGELOG.md

```diff
@@ -1,9 +1,16 @@
 # CHANGELOG - Riak_Repl
 
+## 1.0.0
+
+ * [FEATURE] adds realtime_queue_stats metrics for consumers
+ * [FEATURE] adds realtime source/sink connection stats
+ * [FEATURE] adds fullsync_coordinator stats
```

Comment:
We need to explicitly state the metrics name change here

Reply:
i.e. list all the new metrics in the CHANGELOG?

Reply:
No, state that "riak_repl.fullsync_coordinator.<cluster>.<key>" metrics  are now "riak_repl.fullsync_coordinator.<key>" with a cluster tag